gcloud/pubsub/test_subscription.py

```diff
@@ -620,29 +603,68 @@ def test_check_iam_permissions_w_alternate_client(self):
 
         self.assertEqual(len(allowed), 0)
         self.assertEqual(len(conn1._requested), 0)
-        self.assertEqual(len(conn2._requested), 1)
-        req = conn2._requested[0]
-        self.assertEqual(req['method'], 'POST')
-        self.assertEqual(req['path'], PATH)
-        self.assertEqual(req['data'], REQUESTED)
+        self.assertEqual(len(conn2._requested), 0)
+        self.assertEqual(conn2._tested_iam_permissions,
+                         (self.SUB_PATH, ROLES))
 
 
-class _Connection(object):
+class _Connection(object):  # pylint: disable=too-many-instance-attributes
```

Comment:
Ick on the disable. What is the limit and how far does this code exceed it.

Can you just add an [over-ride][1]?

[1]: https://github.com/GoogleCloudPlatform/gcloud-python/blob/ac50331bebc0fa92071d7e4fe9bdd66b139f1964/scripts/run_pylint.py#L61-L70